kview/src/main/java/dev/ikm/komet/kview/mvvm/view/navigation/PatternNavEntryController.java

```diff
@@ -252,7 +275,20 @@ protected void updateItem(Object item, boolean empty) {
 
                         setContentDisplay(ContentDisplay.GRAPHIC_ONLY);
                         label.setText(entityDescriptionText);
-                        tooltip.setText(entityDescriptionText);
+
+                        final String finalEntityDescriptionText = entityDescriptionText;
+
+                        // run later on the application thread to allow rendering to complete
+                        // which is needed to know if the label text is truncated
+                        Platform.runLater(() -> {
```

Comment:
We don't need a platform.runlater here.
tip: calling platform.runlater when a platform.runlater is not required can decrease perfromance.